### PR TITLE
Add README.md for relaynode

### PR DESCRIPTION
**Pull request by lochjin**

### CODE REVIEW

File: cmd/relaynode/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+```bash
+~ cd ./cmd/relaynode
+~ go build
+~ ./relaynode -h
+```
+## Getting Started
+
+* Start `relaynode`
+```bash
+~ ./relaynode
```
Comment (forchain) [inline cmd/relaynode/README.md:21]:
```sh
tony@dagfans ~/github.com/qitmeer/qitmeer/cmd/relaynode (master-p2p*) $ ./relaynode
2020-11-17|11:09:22.011 [INFO ] Start relay node...                 module=RELAY
2020-11-17|11:09:22.062 [INFO ] Load config completed               module=RELAY
2020-11-17|11:09:22.062 [INFO ] NetWork:mixnet  Genesis:119dbc54464156d342c6bc4655d10c86ce9be002cd8eac9bd83f5d2bbe3e0b4c module=RELAY
2020-11-17|11:09:22.062 [INFO ] Run relay node...                   module=RELAY
2020-11-17|11:09:22.071 [ERROR] Failed to create host %v            module=RELAY LOG15_ERROR= LOG15_ERROR="Normalized odd number of arguments by adding nil"
2020-11-17|11:09:22.076 [INFO ] Stop relay node                     module=RELAY
2020-11-17|11:09:22.076 [ERROR] failed to listen on any addresses: [listen tcp4 0.0.0.0:1001: bind: permission denied listen tcp4 103.1.154.227:1001: bind: permission denied] module=RELAY
```

**Review by forchain — CHANGES_REQUESTED** [review]
failed to listen on any addresses

File: cmd/relaynode/README.md (hunk 1 above)
Comment (forchain) [reply cmd/relaynode/README.md:21]: sudo is OK

```sh
tony@dagfans ~/github.com/qitmeer/qitmeer/cmd/relaynode (master-p2p*) $ sudo ./relaynode
2020-11-17|11:17:40.701 [INFO ] Start relay node...                 module=RELAY
2020-11-17|11:17:40.743 [INFO ] Load config completed               module=RELAY
2020-11-17|11:17:40.744 [INFO ] NetWork:mixnet  Genesis:119dbc54464156d342c6bc4655d10c86ce9be002cd8eac9bd83f5d2bbe3e0b4c module=RELAY
2020-11-17|11:17:40.744 [INFO ] Run relay node...                   module=RELAY
2020-11-17|11:17:40.751 [INFO ] Relay Address: /ip4/103.1.154.227/tcp/1001/p2p/16Uiu2HAmA1vRXRU4witxhXouDtLQdCCNfHaJXcQFG9jrtr8aXsXv
 module=RELAY
2020-11-17|11:17:40.751 [INFO ] You can copy the relay address and configure it to the required Qitmeer-Node module=RELAY
```

Comment (forchain) [reply cmd/relaynode/README.md:21]: The default port 1001 is within the system service reserved range (1~1024), please change it greater to work around  the sudo requirement

Comment (lochjin) [reply cmd/relaynode/README.md:21]: https://github.com/Qitmeer/qitmeer/pull/396/commits/c748f021a14035e3c5ce39ee19375ed3c1c53017

Comment (forchain) [reply cmd/relaynode/README.md:21]: > c748f02

verified

File: cmd/relaynode/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+```
+
+* Copy `Relay Address` from relaynode log.
+
+* Configure startup parameters for Qitmeer nodes in the private network.
+```bash
+~ ./qitmeerd --relaynode=[Relay Address]
```
Comment (forchain) [inline cmd/relaynode/README.md:28]: the binary name of Qitmeer node should be qitmeer rather than Qitmeer

**Review by forchain — APPROVED** [review]
all verified

**Review by dindinw — APPROVED** [review]

**Pull request merged**